### PR TITLE
[2.4] Backport upgrade to k8s 1.18

### PR DESCRIPTION
**Pull request by aiyengar2**
Backports https://github.com/rancher/steve/commit/420f62f642eb4ffc09732af09cbd0846e2a36a25 and https://github.com/rancher/steve/commit/7e84f96c0f3ff8fb11e5ec922d67396e2324246f

Dependent on: https://github.com/rancher/wrangler/pull/93

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,32 +2,32 @@ module github.com/rancher/steve
...
 	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gorilla/mux v1.6.2
 	github.com/gorilla/websocket v1.4.0
 	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/rancher/dynamiclistener v0.2.1-0.20200213165308-111c5b43e932
-	github.com/rancher/wrangler v0.5.3
-	github.com/rancher/wrangler-api v0.5.0
+	github.com/rancher/wrangler v0.5.4-0.20200326191509-4054411d9736
```
Comment (aiyengar2) [inline go.mod:16]: Need to update this after merging https://github.com/rancher/wrangler/pull/93 with the commit from the release/v2.4 branch

**Comment by aiyengar2** [comment]
Last commit (`vendor change`) rolls back changes that were absorbed from master as part of the initial vendor changes to what is available on the 2.4 release and points the go.mod to the 2.4 release branch.

**Comment by ibuildthecloud** [comment]
@aiyengar2 Seems like there should be a wrangler-api PR for this backporting.  Is there some reason it is not needed?

**Comment by aiyengar2** [comment]
@ibuildthecloud wrangler-api only has a master branch, so there's no branch to backport it to. The cherry-picked PR directly references the commit on the master branch.

**Pull request merged**